extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_licensingAndVertical.php

```diff
@@ -1,3 +1 @@
-<div class="wds-global-footer__bottom-bar">
-	<div class="wds-global-footer__bottom-bar-row wds-has-padding"><?= DesignSystemHelper::renderText( $model['description'] ) ?></div>
-</div>
+<div class="wds-global-footer__bottom-bar-row wds-has-padding"><?= DesignSystemHelper::renderText( $model['description'] ) ?></div>
```

Comment:
Why did you remove the wrapper div?

Reply:
I moved this wrapper to `DesignSystemGlobalFooter_index.php`, because I wanted to nest both licensingAndVertical template and mobileSiteButton template inside it.